packages/stryker/src/transpiler/CoverageInstrumenterTranspiler.ts

```diff
@@ -24,7 +24,7 @@ export default class CoverageInstrumenterTranspiler implements Transpiler {
     this.log = getLogger(CoverageInstrumenterTranspiler.name);
   }
 
-  transpile(files: File[]): TranspileResult {
+  public async transpile(files: File[]): Promise<TranspileResult> {
```

Comment:
i think `async` is not needed here. It is only used to mark methods in which you can use `await`